_maps/map_files/debug/runtimestation.dmm

```diff
@@ -1935,7 +1935,7 @@
 /area/construction)
 "gz" = (
 /obj/structure/table,
-/obj/item/card/id/tier5/captains_spare,
+/obj/item/card/id/advanced/gold/captains_spare,
```

Comment:
do we actually still need the 'advanced/' namespacing everywhere or could this be done without, id be OK with putting some effort into doing the substitution for that